includes/class-amp-rest-api.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * AMP REST API.
+ *
+ * @package AMP
+ * @since   2.0
+ */
+
+/**
+ * Class AMP_REST_API.
+ */
+class AMP_REST_API {
+
+	/**
+	 * Init.
+	 */
+	public static function init() {
+		if ( ! class_exists( 'AMP_REST_API' ) ) {
```

Comment:
That check is unneeded. You're already within the `AMP_REST_API` class here, so you can be pretty certain that the class exists :-)

Reply:
Gone :)